### PR TITLE
Use 'externals' field in webpack file, use external dependencies, suggest CDN.

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
Subsumes #159.

@sandersn @RyanCavanaugh @mhegazy

### CODE REVIEW

File: pages/quick-start/react-webpack.md [hunk 1]
```diff
@@ -140,11 +140,22 @@ Create a file at the root of `proj` named `index.html` with the following conten
     </head>
     <body>
         <div id="example"></div>
+
+        <!-- Dependencies -->
+        <script src="./node_modules/react/dist/react.js" />
```
Comment (mhegazy) [inline pages/quick-start/react-webpack.md]: should that be a CDN somewhere?

Comment (DanielRosenwasser) [inline pages/quick-start/react-webpack.md]: It doesn't matter where you get it from. I explain that below.

Comment (mhegazy) [inline pages/quick-start/react-webpack.md]: oh.. i see the comment below.

Comment (mhegazy) [inline pages/quick-start/react-webpack.md]: can not we add the CDN reference, and then tell them they can copy from node_modules instead?

what is the standard react tutorial does?

**Comment by mhegazy** [comment]
:+1:

**Pull request merged**